Dockerfile.multichain

```diff
@@ -19,9 +19,19 @@ RUN sed -i 's#target-dir = "../target"#target-dir = "target"#' .cargo/config.tom
 RUN cargo build --release --package mpc-node
 
 FROM debian:stable-slim as runtime
-RUN apt-get update && apt-get install --assume-yes libssl-dev ca-certificates curl
+RUN apt-get update && apt-get install --assume-yes libssl-dev ca-certificates curl redis-server
+
 RUN update-ca-certificates
 COPY --from=builder /usr/src/app/target/release/mpc-node /usr/local/bin/mpc-node
+
+# Create a script to start both Redis and the Rust app
+RUN echo "#!/bin/bash\nservice redis-server start ./redis.conf\nexec mpc-node" > /start.sh \
+    && chmod +x /start.sh
+
 WORKDIR /usr/local/bin
 
-ENTRYPOINT [ "mpc-node" ]
\ No newline at end of file
+# Expose Redis port (6379) and mpc node's port
+EXPOSE 6379 8080
```

Comment:
hmm, how come we didn't need to expose the node's port before

Reply:
Also wondering why 8080 is now exposed. Port is set at 3000 for the mpc-node service by terraform. we will need to expose 6379 though for redis.